README.md

```diff
@@ -0,0 +1,16 @@
+# git-credential-fdoss
+
+`git-credential-fdoss` is a [git credential helper] that uses the
+[freedesktop.org Secret Service] via [D-Bus] for storage of git credentials.
+
+[git credential helper]: https://git-scm.com/doc/credential-helpers
+[freedesktop.org Secret Service]: https://specifications.freedesktop.org/secret-service/
+[D-Bus]: https://www.freedesktop.org/wiki/Software/dbus/
+
+    go install foxygo.at/git-credential-helper@latest
```

Comment:
I just ran this locally and got:
```
go: foxygo.at/git-credential-helper@latest: unrecognized import path "foxygo.at/git-credential-helper": reading https://foxygo.at/git-credential-helper?go-get=1: 404 Not Found
```

Reply:
Ah yeah. I think I need to add some config to our foxygo.at repo

Reply:
I assume it also will not work until this PR is merge too